stdlib/2.7/typing.pyi

```diff
@@ -174,6 +175,15 @@ class ValuesView(MappingView, Iterable[_VT_co], Generic[_VT_co]):
     def __contains__(self, o: object) -> bool: ...
     def __iter__(self) -> Iterator[_VT_co]: ...
 
+class ContextManager(Generic[_T]):
```

Comment:
It is in the PEP 484, so I tough it was for any version; why having it only for 3.6 anyway?
In this case, it way to allow having `tarfile` in `2and3`.